src/components/notifications/GtxMessaging.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div class="mb-3">
+        <label for="gtxmessaging-api-key" class="form-label">{{ $t("gtxMessagingApiKey") }}</label>
```

Comment:
A general translation key already exists for this

```suggestion
        <label for="gtxmessaging-api-key" class="form-label">{{ $t("API Key") }}</label>
```